models.go

```diff
@@ -19,18 +18,28 @@ type Categories struct {
 type Sub_Categories struct {
 	Sub_category_id   int
 	Sub_category_name string
-	Parent_id         int
+	Parent_id       int
 	Torrents          []Torrents `gorm:"ForeignKey:sub_category_id;AssociationForeignKey:sub_category_id"`
 }
 
+type Statuses struct {
+	Status_id	int
+	Status_name	string
+	Torrents	[]Torrents   `gorm:"ForeignKey:status_id;AssociationForeignKey:status_id"`
+}
+
 type Torrents struct {
-	gorm.Model
 	Id              int            `gorm:"column:torrent_id"`
 	Name            string         `gorm:"column:torrent_name"`
 	Category_id     int            `gorm:"column:category_id"`
 	Sub_category_id int            `gorm:"column:sub_category_id"`
-	Status          int            `gorm:"column:status_id"`
+	Status_id	int	       `gorm:"column:status_id"`
 	Hash            string         `gorm:"column:torrent_hash"`
+	Date		int	       `gorm:"column:date"`
+	Downloads	int	       `gorm:"column:downloads"`
+	Filesize	string	       `gorm:"column:filesize"`
```

Comment:
Unfortunately the current db has them all stored as text, at some point that will all have to be converted for proper sorting.